secedgar/filings/filing_extractor.py

```diff
@@ -6,9 +6,6 @@
 
 from secedgar.utils import make_path
 
-# Pulled from https://github.com/sec-edgar/sec-edgar/blob/
-# dda43dd3b4d1ea19abfe71596b165e22625357c6/SECEdgar/extractor/EDGARExtractor.py
```

Comment:
this reminds me that in the docs it should say that this is an experimental feature and doesn't work 100% of the time, it didnt correctly extract some metadata in my testing.

Reply:
Ok, good to know. We will mark it as experimental.